lib/browser_sniffer/patterns.rb

```diff
@@ -45,6 +45,9 @@ class BrowserSniffer
   REGEX_MAP = {
     :browser => [
       [
+        # Shopify POS Go
+        /WSC6X/i
```

Comment:
They are, but we may not be, since analytics relies on this value as is we were considering hardcoding it to stay the same in the user agent...
Don't remember what decision we made, but until we make the change in our UA or at least test that the ROM level change will actually even effect the value we fetch for the UA...
Let's sync on this.

Reply:
@LeoKaliazine Not sure if I follow. The UA identified here will be WSC6X and WTH1X, so there should be no impact on Mode (as the changes there are independent from this one)

Reply:
Oh sorry, was focused more on the comment than the code, yeah having both works 👍🏻